### ISSUE TITLE
machines: Create new VM dialog is missing a label from the options checkboxes

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#15526]
The create VM dialog implies that the install and run options are part of the memory settings.

![Screenshot from 2021-03-15 11-50-21 memory options](https://user-images.githubusercontent.com/10246/111170286-0d590280-85a4-11eb-8590-05a9fb48a8e0.png)

The group of options should have their own label.

This is both a UX and accessibility issue, thankfully with a simple fix.

**Comment by KKoukiou** [comment]
@garrett the checkboxes have labels on the right. It's not an accessibility issue.

In the PF4 documentation they use checkboxes without the form field labels as well. https://www.patternfly.org/v4/components/form/#horizontal

**Comment by KKoukiou** [comment]
@garrett plus we follow that pattern in other places in cockpit. 
![Screen Shot 2021-03-15 at 15 52 37](https://user-images.githubusercontent.com/14921356/111172904-82c5d280-85a6-11eb-873b-792a2c1ab3c1.png)

**Comment by garrett** [comment]
> @garrett the checkboxes have labels on the right. It's not an accessibility issue.

No, they don't have context, so it looks _and sounds_ like it's part of memory. That's the issue.

> In the PF4 documentation they use checkboxes without the form field labels as well. https://www.patternfly.org/v4/components/form/#horizontal

They don't necessarily.

Those checkboxes could be described as part of "your experience", I guess? If not, then the demo is wrong.

Our checkboxes are definitely not part of memory, even though it looks that way.

**Comment by garrett** [comment]
@garrett plus we follow that pattern in other places in cockpit.

No we don't... at least in that screenshot. The IPv4 network and mask are part of IP configuration.

Oh. I guess you mean DHCP range? That's wrong if it's not part of the mask.
